src/components/qr-reader/qr-reader.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import { QrReader } from "react-qr-reader";
+import { CameraContainer } from "../camera/camera.styles";
+
+export function QRCodeScanner() {
+  const [data, setData] = useState("No Result");
+  return (
+    <CameraContainer className="d-flex text-center bg-alpha">
+      <div
+        style={{
```

Comment:
Can you use a styled component for this instead of inline styles? Also I'm unsure if a constant 400px width will work-- for mobile devices, screen sizes go down to 320px, so it will need to be responsive for that